.github/workflows/docs.yml

```diff
@@ -0,0 +1,41 @@
+name: Rust CI 
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  # TODO remove docs publishing, when crate is published to crates.io
```

Comment:
I have created the issue for our reference and tracking https://github.com/NethermindEth/condor-rs/issues/24

Reply:
Super! Thanks a lot! 👍